NuRadioReco/detector/detector.py

```diff
@@ -94,7 +94,13 @@ def buffer_db(in_memory, filename=None):
                                 'adc_time_delay': channel['adcs.time_delay'],
                                 'adc_nbits': channel['adcs.nbits'],
                                 'adc_n_samples': channel['adcs.n_samples'],
-                                'adc_sampling_frequency': channel['adcs.sampling_frequency']})
```

Comment:
this will likely break the database buffering as these fields are not present at the moment in the mysql database. I actually think that you don't need to change this function at all. We won't use mysql for RNO-G, and for ARIANNA we don't need this functionality. For a simulation study we anyway write the json file manually

Reply:
I can remove that, no problem.